inference-engine/tools/compile_tool/main.cpp

```diff
@@ -113,9 +113,6 @@ static bool parseCommandLine(int *argc, char ***argv, InferenceEngine::Core& ie)
 
     if (std::string::npos != FLAGS_d.find("MYRIAD")) {
         std::vector<std::string> myriadDeviceIds = ie.GetMetric("MYRIAD", METRIC_KEY(AVAILABLE_DEVICES));
-        if (myriadDeviceIds.empty()) {
```

Comment:
reason : When tool doesn't see any MYRIAD devices, it compiles model anyway without runtime_error.